lib/dispatcher.lib.php

```diff
@@ -112,7 +112,7 @@ public function findServer() {
 			}
 			return null;
 		} else {
-			$server =& $PokemonServers[$serverid];
+			$server = &$PokemonServers[$serverid];
```

Comment:
is this change necessary?

Reply:
Nope. Entirely just a nitpick I saw while I was digging around.